src/ros/thread/PublishImagesThread.cpp

```diff
@@ -23,7 +23,7 @@ PublishImagesThread::PublishImagesThread(const Parameters::PublishParameters& pa
 void
 PublishImagesThread::run()
 {
-    rclcpp::Rate rate(m_parameters.fps);
+    const int rate = 1000 / m_parameters.fps;
```

Comment:
Find a better way for rate calculation, this one is unprecise

Reply:
Wall timer creation does not allow to publish using float values, so use microseconds with an int value. Still not optimal, but better than before